GenericDiffractometer.py

```diff
@@ -289,9 +286,10 @@ def init(self):
             if self.transfer_mode is None or self.transfer_mode == "SAMPLE_CHANGER":
                 self.use_sc = True
 
-
         # centring motors
         self.centring_motors_list = ["phiz", "phiy","sampx","sampy"]
+        queue_model_objects.CentredPosition.\
```

Comment:
Yes, you are right. Thanks!

From: Ivars Karpics [mailto:notifications@github.com]
Sent: Friday, April 15, 2016 9:50 AM
To: mxcube/HardwareObjects
Cc: Jie Nan
Subject: Re: [mxcube/HardwareObjects] add zoom related signals (#35)


In GenericDiffractometer.py<https://github.com/mxcube/HardwareObjects/pull/35#discussion_r59837951>:

>          # centring motors

>          self.centring_motors_list = ["phiz", "phiy","sampx","sampy"]

> +        queue_model_objects.CentredPosition.\

The list of centring motors should come from xml (line 238), because each site has a different motors used for centring. If directly method set_diffractometer_motor_names is called then this should be generic.
I will accept this PR and fix this

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/mxcube/HardwareObjects/pull/35/files/274d91816bbd357344e1f07a82f4087747de9081#r59837951>